### DISCUSSION TITLE
ASP.Net Core Web API Learning Guide

### DISCUSSION THREAD

**Discussion by gitfullstacker** [community/community#78283] [Programming Help]
### ASP.Net Core Web API for .Net 8.0

Question

### Body

**Hello!**
I'm currently enrolled in a **Development course** led by renowned instructors. I'm looking for a helpful repository on Git that I can use as a reference for my lessons. Thank you!

**Comment by Akidul-islam** [comment]
you might find these GitHub repositories helpful:

Microsoft's Official Samples:

Repository: [aspnet/AspNetCore](https://github.com/aspnet/AspNetCore)
Look for the "samples" directory, which contains various sample projects, including those for Web API.
AspNetCoreWorkshop:

Repository: [dotnet-presentations/aspnetcore-workshop](https://github.com/dotnet-presentations/aspnetcore-workshop)
This workshop-style repository provides hands-on exercises and guidance for learning ASP.NET Core, including Web API.
AspNetCore.Docs:

Repository: [docs/aspnetcore](https://github.com/dotnet/AspNetCore.Docs)
Check the documentation for ASP.NET Core Web API, and you can find code samples and explanations
